server/src/main/java/org/elasticsearch/cluster/routing/allocation/Explanations.java

```diff
@@ -32,7 +32,7 @@ public static final class Allocation {
         public static final String NO_COPIES = """
             Elasticsearch can't allocate this shard because there are no copies of its data in the cluster. Elasticsearch will allocate \
             this shard when a node holding a good copy of its data joins the cluster. If no such node is available, restore this index \
-            from a recent snapshot.""";
+            from a recent snapshot; for more information, see:\s""" + org.elasticsearch.common.ReferenceDocs.ALLOCATION_EXPLAIN_NO_COPIES;
```

Comment:
Nit: we're writing in full sentences here, no need for the semicolon

```suggestion
            from a recent snapshot. For more information, see \s""" + org.elasticsearch.common.ReferenceDocs.ALLOCATION_EXPLAIN_NO_COPIES;
```